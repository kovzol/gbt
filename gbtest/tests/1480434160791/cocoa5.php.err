0.00user 0.00system 0:00.01elapsed 80%CPU (0avgtext+0avgdata 10168maxresident)k
0inputs+16outputs (0major+707minor)pagefaults 0swaps
