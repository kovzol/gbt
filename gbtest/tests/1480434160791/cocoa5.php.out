*** Cannot access default package directory: "packages" ***

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!! WARNING: Standard CoCoA packages not found !!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

   ______      ______      ___         ______
  / ____/___  / ____/___  /   |       / ____/
 / /   / __ `/ /   / __ `/ /| |______/___ `  
/ /___/ /_/ / /___/ /_/ / ___ /_____/___/ /  
`____/`____/`____/`____/_/  |_|    /_____/   
With CoCoALib and external libraries CDD FROBBY GFAN GSL NORMALIZ READLINE 
indent(VersionInfo()); -- for information about this version

# use QQ[v26,v24,v25,v23,v20,v18,v19,v16,v17,v15,v10,v9,v8,v7,v6,v5,v4,v3,v2,v1<v18,v19,v16,v17,v15,v10,v9,v8,v7,v6,v5,v4,v3,v2,v1,                         v13,v14], DegRevLex;

# GBasis(ideal(-1*v9,-1*v7,-1+v10,v8,-1+v5,v3,v6,v4,v25^2*v24^2+-1*v26^2*v23^2,<+v10,v8,-1+v5,v3,v6,v4,v25^2*v24^2+-1*v26^2*v23^2,-                         1*v18+v16+v5+-1*v3,-1*v17<^2*v24^2+-1*v26^2*v23^2,-1*v18+v16+v5+-1*v3,-1*v17+                         v15+-1*v6+v4,v19*v18+-1*v<*v18+v16+v5+-1*v3,-1*v17+v15+-1*v6+v4,v19*v18+-1*v2                         0*v17+-1*v19*v16+v17*v16+<15+-1*v6+v4,v19*v18+-1*v20*v17+-1*v19*v16+v17*v16+v                         20*v15+-1*v18*v15,-1*v6+v<*v17+-1*v19*v16+v17*v16+v20*v15+-1*v18*v15,-1*v6+v4                         +-1*v5*v4+v6*v3,-1*v19*v6<0*v15+-1*v18*v15,-1*v6+v4+-1*v5*v4+v6*v3,-1*v19*v6+                         v20*v5+v19*v4+-1*v5*v4+-1<-1*v5*v4+v6*v3,-1*v19*v6+v20*v5+v19*v4+-1*v5*v4+-1*                         v20*v3+v6*v3,v24^2+-1*v16<20*v5+v19*v4+-1*v5*v4+-1*v20*v3+v6*v3,v24^2+-1*v16^                         2+-1*v15^2+2*v16*v14+-1*v<20*v3+v6*v3,v24^2+-1*v16^2+-1*v15^2+2*v16*v14+-1*v1                         4^2+2*v15*v13+-1*v13^2,-1<+-1*v15^2+2*v16*v14+-1*v14^2+2*v15*v13+-1*v13^2,-1+                         v25^2+-1*v20^2+2*v19+-1*v<^2+2*v15*v13+-1*v13^2,-1+v25^2+-1*v20^2+2*v19+-1*v1                         9^2,-1*v5*v4+v6*v3+v5*v2+<25^2+-1*v20^2+2*v19+-1*v19^2,-1*v5*v4+v6*v3+v5*v2+-                         1*v3*v2+-1*v6*v1+v4*v1,v2<^2,-1*v5*v4+v6*v3+v5*v2+-1*v3*v2+-1*v6*v1+v4*v1,v23                         ^2+-1*v16^2+-1*v15^2+2*v1<*v3*v2+-1*v6*v1+v4*v1,v23^2+-1*v16^2+-1*v15^2+2*v16                         *v2+-1*v2^2+2*v15*v1+-1*v<2+-1*v16^2+-1*v15^2+2*v16*v2+-1*v2^2+2*v15*v1+-1*v1                         ^2,-1*v9*v8+v10*v7+v9*v2+<v2+-1*v2^2+2*v15*v1+-1*v1^2,-1*v9*v8+v10*v7+v9*v2+-                         1*v7*v2+-1*v10*v1+v8*v1,v<2,-1*v9*v8+v10*v7+v9*v2+-1*v7*v2+-1*v10*v1+v8*v1,v2                         6^2+-1*v20^2+-1*v19^2+2*v<*v7*v2+-1*v10*v1+v8*v1,v26^2+-1*v20^2+-1*v19^2+2*v2                         0*v2+-1*v2^2+2*v19*v1+-1*<^2+-1*v20^2+-1*v19^2+2*v20*v2+-1*v2^2+2*v19*v1+-1*v                         1^2,1+-1*v16^2+-1*v15^2+2<*v2+-1*v2^2+2*v19*v1+-1*v1^2,1+-1*v16^2+-1*v15^2+2*                         v16*v2+-2*v1+2*v15*v1,-1*<^2,1+-1*v16^2+-1*v15^2+2*v16*v2+-2*v1+2*v15*v1,-1*v                         15*v14+v16*v13+v15*v2+-1*<16*v2+-2*v1+2*v15*v1,-1*v15*v14+v16*v13+v15*v2+-1*v                         13*v2+-1*v16*v1+v14*v1));
[v3, v4, v5 -1, v6, -v7, v8, -v9, v10 -1, -v17 +v15, -v18 +v16 +1, v2, v20, -v1, -v16^2 -v15^2 +1, v19 -v15, v23^2 -1, v25^2 -v15^2 +2*v15 -1, v24^2 +2*v15*v13 -v13^2 +2*v16*v14 -v14^2 -1, v26^2 -v15^2, v16*v13 -v15*v14, -2*v15^2*v13 -v15*v14^2 -2*v15^2 -2*v16*v14 +v14^2 -v15 +2*v13 +1, -2*v15*v13^2 -2*v15*v14^2 -4*v15*v13 +2*v13^2 -2*v16*v14 +v14^2 -2*v15 +4*v13 +1, 2*v16*v15*v14 -v15*v14^2 -2*v15^2 -2*v16*v14 +v14^2 -v15 +1, 2*v15^2*v14 -2*v15*v13*v14 -v16*v14^2 +2*v16*v15 -4*v15*v14 +2*v13*v14 -v16 +2*v14, 2*v15*v13*v14^2 +v14^4 +6*v15*v14^2 -2*v13*v14^2 +2*v15*v13 -4*v13^2 +4*v16*v14 -4*v14^2 +2*v15 -6*v13 -1, -2*v16*v14^3 +v14^4 +2*v15*v14^2 +4*v15*v13 -4*v13^2 +2*v16*v14 +2*v15 -4*v13 -1, 2*v15*v14^4 -v13*v14^4 +v14^4 +4*v13^3 +8*v15*v14^2 -2*v13*v14^2 +8*v15*v13 -4*v13^2 +8*v16*v14 -6*v14^2 +6*v15 -13*v13 -3, -v13^2*v14^4 -v14^6 +4*v13^4 +2*v13^2*v14^2 +2*v14^4 +8*v13^3 +3*v13^2 -v14^2]

# 